### PR TITLE
change(docs): Add changelog entry for stable mining feature

### PR DESCRIPTION
**Pull request by teor2345**
## Motivation

We want to talk about stabilising mining RPCs in the changelog for the next release.

The mining part of the README also needs an update.

This is a follow-up to ticket #7361.

## Review

This is a low priority documentation change. Technically it blocks the next release, but it should be merged long before then.

### Reviewer Checklist

  - [ ] Are the PR labels correct?
  - [ ] Does the code do what the ticket and PR says?
    - [ ] Does it change concurrent code, unsafe code, or consensus rules?
  - [ ] How do you know it works? Does it have tests?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,26 @@ All notable changes to Zebra are documented in this file.
...
+
+Zebra's mining RPCs are now available in release builds. TODO: rest of intro
+
+This release contains the following changes:
+ 
+### Mining RPCs in Production Builds
+
+Zebra's mining RPCs are now available in release builds (#7740). Any Zebra instance can be used
+by a miner or mining pool. This stabilises 12 RPCs, including  `getblocktemplate`, `submitblock`,
+`getmininginfo`, `getnetworksolps`, `[z_]validateaddress` and `getblocksubsidy`. For more infomation,
```
Comment (upbqdn) [inline CHANGELOG.md]:
```suggestion
`getmininginfo`, `getnetworksolps`, `[z_]validateaddress` and `getblocksubsidy`. For more information,
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,26 @@ All notable changes to Zebra are documented in this file.
...
+## [Zebra 1.4.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.4.0) - TODO: DATE
+
+Zebra's mining RPCs are now available in release builds. TODO: rest of intro
+
+This release contains the following changes:
+ 
+### Mining RPCs in Production Builds
+
+Zebra's mining RPCs are now available in release builds (#7740). Any Zebra instance can be used
+by a miner or mining pool. This stabilises 12 RPCs, including  `getblocktemplate`, `submitblock`,
```
Comment (upbqdn) [inline CHANGELOG.md]:
```suggestion
by a solo miner or mining pool. This stabilises 12 RPCs, including  `getblocktemplate`, `submitblock`,
```

**Review by upbqdn — DISMISSED** [review]
I wonder if we should also mention that miners will still need to run `zcashd` for rewards distribution.

**Comment by teor2345** [comment]
> I wonder if we should also mention that miners will still need to run `zcashd` for rewards distribution.

I think it depends on the mining software. Some of them make their own coinbase transactions that distribute rewards, or using their own miner address.

What do you think about commit [44a5f9f](https://github.com/ZcashFoundation/zebra/pull/7788/commits/44a5f9fcdad020fb03da2712354b73a28c535679)?
(The README needed an update for mining.)

We could copy some of that wording into the CHANGELOG?

**Review by arya2 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -35,10 +35,10 @@ them:
...
 - To generate transactions, [run Zebra with
   `lightwalletd`](https://zebra.zfnd.org/user/lightwalletd.html).
-- To generate blocks, [enable mining
-  support](https://zebra.zfnd.org/user/mining.html), and use a mining pool or
-  miner with Zebra's mining JSON-RPCs. Mining support is currently incomplete,
-  experimental, and off by default.
+- To generate blocks, use a mining pool or miner with Zebra's mining JSON-RPCs.
+  Currently Zebra can only send mining rewards to a single fixed address.
+  To distribute rewards, use mining software that creates its own distribution transactions,
+  or a light wallet or `zcashd` wallet.
```
Comment (arya2) [inline README.md]: Nitpick/Optional:

```suggestion
  To distribute rewards, use mining software that creates its own distribution transactions,
  a light wallet or the `zcashd` wallet.
```

**Review by upbqdn — APPROVED** [review]

**Pull request merged**